src/app/builders/components/Table/RewardsCell.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Paragraph } from '@/components/TypographyNew'
+import { DottedUnderlineLabel } from '@/app/backing/components/DottedUnderlineLabel/DottedUnderlineLabel'
+import { RifRbtcTooltip } from '@/app/backing/components/Popovers/RifRbtcTooltip'
```

Comment:
Please move `RifRbtcTooltip` to `@/app/components/` so that we don't import from a “cousin” path.